ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -0,0 +1,208 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  ashenEyeDirections?: string[];
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheThirdCircle,
+  triggers: [
+    {
+      id: 'P3N Experimental Fireplume 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
+      response: Responses.getInThenOut(),
+    },
+    {
+      id: 'P3N Experimental Fireplume 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6696', source: 'Phoinix', capture: false }),
+      alertText: (_data, _matches, output) => {
+        return output.outOfMiddle!();
+      },
+      outputStrings: {
+        outOfMiddle: {
+          en: 'Out Of Middle',
+          de: 'Raus aus der Mitte',
+          ja: '横へ',
+          cn: '远离中间',
+          ko: '가운데 피하기',
+        },
+      },
+    },
+    {
+      id: 'P3N Scorched Exaltation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B8', source: 'Phoinix', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P3N Heat of Condemnation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '66B2', source: 'Phoinix' }),
+      suppressSeconds: 1,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'P3N Darkened Fire Aoe',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '010[C-F]' }),
```

Comment:
```suggestion
      netRegex: NetRegexes.headMarker({ id: ['010C', '010D', '010E', '010F'] }),
```

Apologies for a tiny nit on something you already updated, but where possible it might be nice to have IDs be individually searchable.

Reply:
I was orienting on triggers like https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/05-shb/raid/e8s.ts#L152 which are searchable with the same method as this one would be

Reply:
Oops, sorry, I forgot that this was still unresolved.  Agreed that E8S doesn't do this, but I think JLGarber was suggesting that in the future it makes life easier for developers to list out ids in full so that grep/ctrl+f style searches for full ids will turn them up more easily.

Reply:
Sorry, yes, I should have worded that a lot more clearly. It's a minimal cost now to write them in expanded form, while it could easily be more difficult to search for them in regex form later. Obviously not worth changing here, but definitely for the future.